app/views/layouts/my_facilities.html.erb

```diff
@@ -1,30 +1,4 @@
 <% content_for(:content) do %>
-  <div class="sub-nav">
-    <div class="container">
-      <div class="px-3 px-lg-0">
-        <%= link_to(my_facilities_overview_path, class: "sub-nav-link #{active_controller_and_action?("my_facilities", "index")}") do %>
-          Overview
-        <% end %>
-        <%= link_to(my_facilities_bp_controlled_path(preserve_query_params(request.query_parameters, ["zone", "size", "facility_group"])), class: "sub-nav-link #{active_action?("bp_controlled")}") do %>
-          BP controlled
-        <% end %>
-        <%= link_to(my_facilities_bp_not_controlled_path(preserve_query_params(request.query_parameters, ["zone", "size", "facility_group"])), class: "sub-nav-link #{active_action?("bp_not_controlled")}") do %>
-          BP not controlled
-        <% end %>
-        <%= link_to(my_facilities_missed_visits_path(preserve_query_params(request.query_parameters, ["zone", "size", "period", "facility_group"])), class: "sub-nav-link #{active_action?("missed_visits")}") do %>
-          Missed visits
-        <% end %>
-        <% if current_admin.drug_stocks_enabled? %>
-          <%= link_to(my_facilities_drug_stocks_path(preserve_query_params(request.query_parameters, ["facility_group", "zone", "size", "for_end_of_month"])), class: "sub-nav-link #{active_action?("drug_stocks")}") do %>
-            Drug stock
-          <% end %>
-          <%= link_to(my_facilities_drug_consumption_path(preserve_query_params(request.query_parameters, ["facility_group", "zone", "size", "for_end_of_month"])), class: "sub-nav-link #{active_action?("drug_consumption")}") do %>
-            Drug consumption
-          <% end %>
-        <% end %>
-      </div>
-    </div>
-  </div>
```

Comment:
Looks like we don't need this layout anymore. Mind if I rip it out?

Reply:
Yes, please tear it out!